.gitignore

```diff
@@ -10,6 +10,7 @@
 /bin/tool/
 /include/
 /lib*/
+/build/
```

Comment:
Since we don't have a convention in naming the build tree anymore, i.e. user can use a subdir name whatever they like for their build tree and can put that subdir wherever they like too (including putting it inside the source tree, like in your case), ignoring just the 'build' subdir is amounted to hard-coding. Either move your build tree out or use the source tree itself as build tree to solve the problem. Alternatively,  we can agree on a naming convention for a build tree and ignore the subdir matching that naming convention. See how 'configure*' shell scripts are being ignored,  for example. In short no hard coding. 

Thanks for the PR. Glad to have a new 2D subsystem maintainer.